src/Aspire.Hosting.Testing/Aspire.Hosting.Testing.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <TargetFrameworks>$(AllTargetFrameworks)</TargetFrameworks>
```

Comment:
I don't think I'm still following why we need to do this (if we disable CPM for the conflicting version as discussed)

Reply:
Disabling CPM has risks and downsides as well. I think if we are going to make that change, we should make it everywhere. Multi-targeting the package is more inline with our current dependency policy.